trace-normalization/src/normalize_utils.rs

```diff
@@ -25,23 +32,138 @@ pub(crate) fn truncate_utf8(s: &str, limit: usize) -> &str {
     s
 }
 
-// NormalizeService returns a span service or an error describing why normalization failed.
-// TODO: Implement this in a future PR
-// pub fn normalize_service(svc: String, lang: String) -> (String, Option<errors::NormalizeErrors>) {
-// if svc == "" {
-//     return (fallback_service(lang), errors::NormalizeErrors::ErrorEmpty);
-// }
-// if svc.len() > MAX_SERVICE_LEN {
-//     return (truncate_utf8(svc, MAX_SERVICE_LEN), errors::NormalizeErrors::ErrorTooLong.into());
-// }
-// TODO: implement tag normalization
-// let s: String = normalize_tag(svc);
-// if s == "" {
-//     return (fallbackService(lang), errors::NormalizeErrors::ErrorInvalid)
-// }
-// return (s, err)
-// (svc, None)
-// }
+// fallback_service returns the fallback service name for a service
+// belonging to language lang.
+// In the go agent implementation, if a lang was specified in TagStats
+// (extracted from the payload header) the fallback_service name would be "unnamed-{lang}-service".
+pub(crate) fn fallback_service() -> String {
+    DEFAULT_SERVICE_NAME.to_string()
+}
+
+// normalize_service normalizes a span service
+pub(crate) fn normalize_service(svc: &str) -> anyhow::Result<String> {
+    anyhow::ensure!(!svc.is_empty(), "Normalizer Error: Empty service name.");
+
+    let truncated_service = if svc.len() > MAX_SERVICE_LEN {
+        truncate_utf8(svc, MAX_SERVICE_LEN)
+    } else {
+        svc
+    };
```

Comment:
You can simplify this to just call `truncate_utf8`, right? The function looks designed to handle this just fine.

```suggestion
    let truncated_service = truncate_utf8(svc, MAX_SERVICE_LEN);
```